pyvera/__init__.py

```diff
@@ -293,13 +307,7 @@ def set_value(self, name, value):
                         'action': 'Set' + name,
                         'newLoadlevelTarget': value}
                 else:
-                    payload = {
```

Comment:
Yes, let's clean this up. Of our current devices `LoadLevelTarget` is used by `VeraDimmer` and `VeraCurtain`.

I'll do some digging to see if I can find what else it might be used for.